src/Components/Error/Error.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './Error.css'
+import errorImg from '../../assets/errorbot.png'
```

Comment:
What do you think about adding semicolons to the end of these import statements?